app/src/main/java/org/simple/clinic/Globals.kt

```diff
@@ -4,3 +4,5 @@ const val MAX_ALLOWED_PATIENT_AGE: Int = 120
 const val MIN_ALLOWED_PATIENT_AGE: Int = 1
 const val SECURITY_PIN_LENGTH: Int = 4
 const val LOGIN_OTP_LENGTH = 6
+
+val REDACTED_STRING = "U+2588".repeat(5)
```

Comment:
We have a global object `org.simple.clinic.util.Unicode` which you can use for defining special unicode sequences like this. Can you move this there please?

Reply:
Ah, I was not aware of that. Moving it.